openadapt/strategies/vanilla.py

```diff
@@ -35,20 +35,20 @@ class VanillaReplayStrategy(strategies.base.BaseReplayStrategy):
     def __init__(
         self,
         recording: models.Recording,
-        replay_instructions: str = "",
+        instructions: str = "",
```

Comment:
@shashank40 if you remove the rest of this PR and only include the changes to this file I will be happy to merge it 👍